Tools/autotest/vehicle_test_suite.py

```diff
@@ -4487,6 +4494,9 @@ def TestLogDownloadMAVProxyCAN(self, upload_logs=False):
             "CAN_D1_UC_S1_PRO": 2,
             })
         self.reboot_sitl()
+
+        self.set_parameter('SIM_SPEEDUP', 1)
```

Comment:
We might need to reboot at the end of the text (after the context pop which reverts this parameter - or we may end up running the next test at speedup 1.... and if I can fix the problem where the reboot seems to make this parameter not take effect then that would persist until the end of the suite.